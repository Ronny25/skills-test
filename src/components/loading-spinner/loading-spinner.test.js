import React from 'react';
import renderer from 'react-test-renderer';

import LoadingSpinner from './loading-spinner';

describe('<LoadingSpinner />', function () {

  it('should renders correctly', () => {
    const tree = renderer
      .create(<LoadingSpinner />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

});
