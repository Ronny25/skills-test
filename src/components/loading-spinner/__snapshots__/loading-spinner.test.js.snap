// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingSpinner /> should renders correctly 1`] = `
<div
  className="css-105nsg"
>
  <div
    className="css-wai5c7"
  >
    <div
      className="sk-cube sk-cube1"
    />
    <div
      className="sk-cube sk-cube2"
    />
    <div
      className="sk-cube sk-cube3"
    />
    <div
      className="sk-cube sk-cube4"
    />
    <div
      className="sk-cube sk-cube5"
    />
    <div
      className="sk-cube sk-cube6"
    />
    <div
      className="sk-cube sk-cube7"
    />
    <div
      className="sk-cube sk-cube8"
    />
    <div
      className="sk-cube sk-cube9"
    />
  </div>
</div>
`;
